ql/src/test/org/apache/hadoop/hive/ql/metadata/TestMaterializedViewsCache.java

```diff
@@ -157,7 +157,7 @@ private Table getTable(String db, String tableName, String definition) {
   private static HiveRelOptMaterialization createMaterialization(Table table) throws ParseException {
     return new HiveRelOptMaterialization(
             new DummyRel(table), new DummyRel(table), null, asList(table.getDbName(), table.getTableName()),
-            EnumSet.allOf(HiveRelOptMaterialization.RewriteAlgorithm.class),
+            EnumSet.allOf(RewriteAlgorithm.class),
```

Comment:
Should we use RewriteAlgorithm.ALL instead?

Reply:
done